prime-router/src/main/kotlin/azure/ActionHistory.kt

```diff
@@ -464,10 +465,10 @@ class ActionHistory(
         // TODO: Need to update this process to have a better way to determine what messages should be sent
         //  automatically as part of queueMessages and what are being send manually as part of the parent function.
         //  The automatic queueing uses the action name as the queue name, and this is not the case for FHIR actions
-        if (event.eventAction != Event.EventAction.BATCH &&
-            event.eventAction != Event.EventAction.DESTINATION_FILTER &&
-            event.eventAction != Event.EventAction.RECEIVER_FILTER &&
-            event.eventAction != Event.EventAction.TRANSLATE
```

Comment:
the simple statement a lot of what's in RS is tightly coupled with the rest of RS in various ways. It was a challenge to find a place where I could cleave out the logic that went into "shared" without dragging a ton of other things along with it. 

EventAction is used by the queue logic that got ported over but Event is not. Moreover, if we migrate Event over we also have include jackson and jooq dependencies as well as other packages like 

```
gov.cdc.prime.router.fhirengine.engine.LookupTableValueSet
org.jooq.Catalog
org.jooq.EnumType
org.jooq.Schema
```

as well as other dependencies and packages because a lot of what's in RS has some kind of connection to other things in RS.

for example, `LookupTableValueSet` brings in 

```
gov.cdc.prime.router.Metadata
gov.cdc.prime.router.fhirengine.translation.hl7.SchemaException
gov.cdc.prime.router.fhirengine.translation.hl7.ValueSetCollection
```

bottom line - it's good we're doing the work to decouple the app and it's going to be challenging to do so cleanly along business logic lines at first until we get more of the app decoupled from itself 😅